static/app/components/events/interfaces/spans/spanTreeModel.tsx

```diff
@@ -44,6 +44,7 @@ class SpanTreeModel {
   fetchEmbeddedChildrenState: FetchEmbeddedChildrenState = 'idle';
   showEmbeddedChildren: boolean = false;
   embeddedChildren: Array<SpanTreeModel> = [];
+  isEmbeddedTransactionTimeAdjusted: boolean = false;
```

Comment:
@0Calories you'll need to add this to `makeObservable()` as an `observable`.

Reply:
Added 🙂